content/intro-to-storybook/vue/es/simple-component.md

```diff
@@ -0,0 +1,277 @@
+
+Este proceso es similar a [Test-driven development](https://en.wikipedia.org/wiki/Test-driven_development) (TDD) al que podemos llamar “[Visual TDD](https://blog.hichroma.com/visual-test-driven-development-aec1c98bed87)”.
+
+## Ajustes iniciales
+
+Primero, vamos a crear el componente Task y el archivo de historias de storybook que lo acompaña: `src/components/Task.js` y `src/components/Task.stories.js`.
+
+Comenzaremos con una implementación básica de `Task`, simplemente teniendo en cuenta los atributos que sabemos que necesitaremos y las dos acciones que puedes realizar con una tarea (para moverla entre las listas):
+
+```html
+<template>
+  <div class="list-item">
+    <input type="text" :readonly="true" :value="this.task.title" />
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'task',
+    props: {
+      task: {
+        type: Object,
+        required: true,
+        default: () => ({}),
+      },
+    },
+  };
+</script>
+```
+
+Arriba, renderizamos directamente `Task` basándonos en la estructura HTML existente de la app Todos.
+
+A continuación creamos los tres estados de prueba de Task dentro del archivo de historia:
+
+```javascript
+import { action } from '@storybook/addon-actions';
+import Task from './Task';
+export default {
+  title: 'Task',
+  // Our exports that end in "Data" are not stories.
+  excludeStories: /.*Data$/,
+};
+export const actionsData = {
+  onPinTask: action('onPinTask'),
+  onArchiveTask: action('onArchiveTask'),
+};
+
+export const taskData = {
+  id: '1',
+  title: 'Test Task',
+  state: 'Task_INBOX',
+  updated_at: new Date(2019, 0, 1, 9, 0),
+};
+
+const taskTemplate = `<task :task="task" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`;
+
+// default task state
+export const Default = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: taskData,
+    },
+  },
+  methods: actionsData,
+});
+// pinned task state
+export const Pinned = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_PINNED',
+      },
+    },
+  },
+  methods: actionsData,
+});
+// archived task state
+export const Archived = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_ARCHIVED',
+      },
+    },
+  },
+  methods: actionsData,
+});
+```
+
+Existen dos niveles básicos de organización en Storybook. El componente y sus historias hijas. Piensa en cada historia como una permutación posible del componente. Puedes tener tantas historias por componente como se necesite.
+
+- **Componente**
+  - Historia
+  - Historia
+  - Historia
+
+Para decirle a Storybook sobre el componente que estamos documentando, creamos una exportación `por defecto` que contiene:
```

Comment:
@IvanSotelo i was reading this and i think that this should be phrased a bit different. In here you translated it but further down in the document you didn't. The reasoning behind this is that this sentence refers to es6 syntax, more specifically `default export` that is used by the `task.story.js` file to tell Storybok the shape and how the story related to that component.

Reply:
@IvanSotelo i saw that you marked this as resolved but the change did not go through, can you share some thoughts on why? Like i said above, this is one of those situations that we cannot translate it "ipsis verbis" as this is a framework/syntax thing and the flow of the documentation tends to lead that way.